docs/learning/backend/Workspace.md

```diff
@@ -202,6 +204,8 @@ Of course `SettingDictionary`s wouldn't be very useful if you could only define
 
 > Hint: iTwin.js supports [JSON5](https://json5.org/) format to permit comments in settings files. [VSCode](https://code.visualstudio.com/) recognizes the `.json5` extension to edit JSON5 content with comments.
 
+### iModel Settings
```

Comment:
Empty section

Reply:
yes, i'm not done with the documentation yet.

Reply:
updated, still more work to do, but i want to complete this PR